app/services/dashboard/audit_service.rb

```diff
@@ -41,19 +42,19 @@ def moab_to_catalog_audit_ok?
 
     def moab_checksum_validation_audit_ok?
       # NOTE: unsure if there needs to be more checking of MoabRecord.status_details for more statuses to figure this out
-      MoabRecord.invalid_checksum.count.zero?
+      MoabRecord.invalid_checksum.annotate(sa).count.zero?
     end
 
     def catalog_to_archive_audit_ok?
-      !ZipPart.where.not(status: 'ok').exists?
+      !ZipPart.where.not(status: 'ok').annotate(sa).exists?
```

Comment:
Can you say more about where in the chain we generally want to place the `annotate` call? Given what's in the README, I expected `annotate` earlier but maybe that's not appropriate here?

Reply:
Great question. I've added this to README:

Note that `.annotate(caller)` should be placed where it receives and returns an `ActiveRecord::Relation`.
```
# Good
ZipPart.where.not(status: 'ok').annotate(caller).count
ZipPart.all.annotate(caller).where.not(status: 'ok').count

# Bad
ZipPart.annotate(caller).where.not(status: 'ok').count
ZipPart.where.not(status: 'ok').count.annotate(caller)
```